src/site/content/en/learn-core-web-vitals/index.njk

```diff
@@ -1,6 +1,56 @@
 ---
-layout: path
+layout: collection
 override:tags: []
 date: 2020-09-16
 pathName: learn-core-web-vitals
+cards:
```

Comment:
This is an example usage of cards on a collection page. Please create a separate example page, somewhere under https://web.dev/handbook/#web.dev-markup, simiar to what @johnrich85  did in https://github.com/GoogleChrome/developer.chrome.com/tree/main/site/en/content-types and render these example cards there, with documentaion on how to use them.

Reply:
I created a document for the usage of cards on a collection page under [web.dev-markup](https://web.dev/handbook/#web.dev-markup).

- https://deploy-preview-9253--web-dev-staging.netlify.app/handbook/markup-cards-collection/ 

I also added the sample page of collection template in the document.

- https://deploy-preview-9253--web-dev-staging.netlify.app/example-collection/